src/extension/background-script/actions/allowances/add.ts

```diff
@@ -28,6 +32,14 @@ const add = async (message, sender) => {
       totalBudget: totalBudget,
       remainingBudget: totalBudget,
       lastPaymentAt: 0,
+      createdAt: Date.now().toString(),
```

Comment:
```suggestion
      createdAt: Date.now(),
```
for the payments we use numbers. 
and I think we should keep the timestamp here as a number.

Reply:
But wouldn't this break for existing users? cc @escapedcat

Reply:
seems we do not save this so far. At least I did not see it and this change is added in this PR

Reply:
Looking at the current code it should be number and won't break things because it was always a number I believe